src/api/status.ts

```diff
@@ -0,0 +1,37 @@
+import { Status, Api, ApiEventEmitter, ModuleProfile } from "../types"
+import { BaseMixinApi, MixinApi } from "./mixin"
+
+// API
+export interface StatusApi {
+  name: 'status',
+  events: {
```

Comment:
should the function `getStatus` be  declared here?

Reply:
Sure, I'll add it.